spec/requests/pages_spec.rb

```diff
@@ -30,5 +30,24 @@
 
       include_examples "policy_views"
     end
+
+    context "viewing a specific privacy policy" do
+      subject do
+        get privacy_policy_path(id: policy.id)
+        response
+      end
+
+      before do
+        allow_any_instance_of(GetIntoTeachingApiClient::PrivacyPoliciesApi).to receive(:get_privacy_policy).with(policy.id).and_return(policy)
```

Comment:
Can we have a test here to ensure the session is set?

Reply:
👍